src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/activities/ActivitiesService.kt

```diff
@@ -71,21 +71,22 @@ class ActivitiesService(
     }
 
   fun updateActivity(event: ScheduleDomainEvent) {
-    val telemetryMap = mutableMapOf("activityScheduleId" to event.additionalInformation.activityScheduleId.toString())
+    val activityScheduleId = event.additionalInformation.activityScheduleId
+    val telemetryMap = mutableMapOf("activityScheduleId" to activityScheduleId.toString())
 
     runCatching {
-      val activitySchedule = activitiesApiService.getActivitySchedule(event.additionalInformation.activityScheduleId)
```

Comment:
switch order so mapping is the first thing checked to be consistent with creates